stake-pool/py/pyproject.toml

```
@@ -0,0 +1,67 @@
+[project]
+name = 'stake-pool'
+version = '1.0'
+authors = [
+  { name= 'joncinque'},
+]
+description = 'solana labs stake pool reference implementation'
+readme = 'README.md'
+license = { file='LICENSE' }
+requires-python = '>=3.7'
+classifiers = [
+    'Programming Language :: Python :: 3',
+    'License :: OSI Approved :: Apache Software License',
+    'Operating System :: OS Independent',
+]
+dependencies = [
+        'anyio==3.3.4',
+        'attrs==21.2.0',
+        'base58==2.1.0',
+        'cachetools==4.2.4',
+        'certifi==2021.10.8',
+        'cffi==1.15.0',
+        'charset-normalizer==2.0.7',
+        'construct==2.10.67',
+        'flake8==4.0.1',
+        'h11==0.12.0',
+        'httpcore==0.13.7',
+        'httpx==0.20.0',
+        'idna==3.3',
+        'iniconfig==1.1.1',
+        'mccabe==0.6.1',
+        'mypy==0.910',
+        'mypy-extensions==0.4.3',
+        'packaging==21.2',
+        'pluggy==1.0.0',
+        'py==1.10.0',
+        'pycodestyle==2.8.0',
+        'pycparser==2.20',
+        'pyflakes==2.4.0',
+        'PyNaCl==1.4.0',
+        'pyparsing==2.4.7',
+        'pytest==6.2.5',
+        'pytest-asyncio==0.16.0',
```

I recently bumped this, sorry!
```suggestion
        'pytest-asyncio==0.19.0',
```